backend/drone.js

```diff
@@ -0,0 +1,60 @@
+const dgram = require("dgram");
+const wait = require("waait");
+const app = require("express")();
+const http = require("http").Server(app);
+const io = require("socket.io")(http);
+const throttle = require("lodash/throttle");
+const commandDelays = require("./commandDelays");
+
+const PORT = 8889;
+const HOST = "192.168.10.1";
+const drone = dgram.createSocket("udp4");
+drone.bind(PORT);
+
+function parseState(state) {
+  return state.split(":").map((x) => x.split(":"));
+}
+
+const droneState = dgram.createSocket("udp4");
+droneState.bind(8890);
+
+drone.on("message", (message) => {
+  console.log(`--> : ${message}`);
+
+  // io.sockets.emit('status', message.toString());
```

Comment:
do you still need this? if not, I'd delete it in the final version :)

Reply:
on the last branch already removed. Sorry for the mess on the lass branch..